Symbols 추가 및 부분 수정

foundy opened the pull request:
* TypeScript Handbook의 Symbols 항목을 추가 했습니다.
* 일부 오타 수정 및 빈줄을 제거 했습니다.

Commit:
Update README.md

* 접속 불가 Handbook URL 수정

Commit:
오타 수정 및 빈줄 삭제

Commit:
Symbols 항목 추가

* introduction 추가
* well-known symbols 추가

trustyoo86 commented:
@foundy 첫번째 PR 갑사합니다 :)
확인해보면서, 몇가지 문장 중에 수정하면 좋을것 같은 내용들을 comment 드리려고 하는데, 검토해보시고 피드백 주시면 반영하겠습니다 :)

Symbol type 문서 추가 감사드립니당!! :)

foundy commented:
@trustyoo86 첫번째 PR이라니 안타깝네요. 😢 
통일성을 갖기 위해 문장 수정은 필요하다고 생각해서 언제든 환영입니다. 😄

Commit:
일부 문장 정정

* 계산된 속성에 computed property 원문 추가
* built-in으로 정정

ES2015 부터 추가된 계산된 속성명(computed property name)의 경우 명확한 한글명이 없으므로
MDN 기준의 문서를 참조하고, 원문을 추가하였습니다.

Reference:
* https://developer.mozilla.org/ko/docs/Web/JavaScript/Reference/Operators/Object_initializer

trustyoo86 commented:
@foundy 반영해주셔서 감사합니다 ㅎㅎ 확실히 영문 documentation을 번역하기가 쉽지는 않네욤 ㅜㅜ merge 하겠습니당!! :) 감사합니다 ㅎㅎ

foundy commented:
@trustyoo86 
computed property name의 경우 번역된 명칭이 일관되지 않고, 매끄러운 느낌이 없어서 원문도 참조하도록 추가하였고, 그 외 추가로 부분 수정했습니다.
정정되었으면 하는 부분 있으면 말씀해주세요~😄

trustyoo86 commented:
@all-contributors please add @foundy for docs

trustyoo86 commented:
@foundy 이미 확인 후 merge 했습니당 :) 감사합니당 ㅋㅋㅋ
